pkg/users/local.js

```diff
@@ -206,7 +206,7 @@ function parse_passwd_content(content) {
             password: column[1],
             uid: parseInt(column[2], 10),
             gid: parseInt(column[3], 10),
-            gecos: column[4],
+            gecos: column[4].split(',')[0],
```

Comment:
I think we need to strip trailing commas here. It's reasonable for someone to have commas in their full name, for example.